.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,60 @@
+name: 🐛 Bug
+description: Create a report to help us improve
+title: "🐛 Bug: "
+labels: ["🐛 Bug", "Status: Triage"]
```

Let's only use `Type: Bug` here.

```suggestion
labels: ["Type: Bug"]
```